examples/ev_automation_keyboard.py

```diff
@@ -0,0 +1,74 @@
+import asyncio
+from mavsdk import System
+
+async def set_params(system, params, announcement):
+    print(announcement)
+
+    async for _ in system.telemetry.landed_state():
+        break
+
+    for param, value in params:
+        await system.param.set_param_int(param, value)
+        await asyncio.sleep(5)
```

Comment:
Why this sleep? That shouldn't be required, right?

Reply:
Unnecessary, removed! Thanks for noticing it!